properties/test_pandas_roundtrip.py

```diff
@@ -110,6 +111,7 @@ def test_roundtrip_pandas_dataframe(df) -> None:
     xr.testing.assert_identical(arr, roundtripped.to_xarray())
 
 
+@pytest.mark.skipif(has_pandas_3, reason="fails to roundtrip on pandas 3")
```

Comment:
```suggestion
@pytest.mark.skipif(has_pandas_3, reason="fails to roundtrip on pandas 3 (we should fix)")
```

Reply:
we do have #9098 to remind us, but we should probably mention that? What do you think about:
```suggestion
@pytest.mark.skipif(
    has_pandas_3,
    reason="fails to roundtrip on pandas 3 (see https://github.com/pydata/xarray/issues/9098)",
)
```